galaxy/api/views/email.py

```diff
@@ -24,13 +24,20 @@
 User = get_user_model()
 
 
+def deny_anonymous_users(user):
+    if user.is_anonymous:
+        raise PermissionDenied()
+
+
 class UserEmailList(base_views.SubListAPIView):
     model = EmailAddress
     serializer_class = serializers.EmailSerializer
     parent_model = User
     relationship = 'emailaddress_set'
 
     def get_queryset(self):
+        deny_anonymous_users(self.request.user)
```

Comment:
I *think* this should be handled in [access.py](https://github.com/ansible/galaxy/pull/1548)

Reply:
Definitely `get_queryset()` is not the best place for checking user permissions.
Please use     `permission_classes = (IsAuthenticated,)`